docs/spfx/sharepoint-subscription-edition-component.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Upgrade components in SharePoint Framework development with SharePoint Server Subscription Edition
+description: SharePoint Server Subscription Edition (SE) supports to ungrade individual components within SharePoint Framework when developing client-side web parts.
+ms.date: 09/08/2023
+ms.localizationpriority: high
+---
+
+
+# Upgrade components in SharePoint Framework development with SharePoint Server Subscription Edition
+
+SharePoint Server Subscription Edition supports SharePoint Framework client-side web parts in classic and modern pages, and extensions in modern pages.
+
+When developing a client-side web part, you can only use the specific versions of components supported by the SharePoint Framework to make the web part compatible with your SharePoint Server.

```

Comment:
Also mention SPFx extensions are those are also supported in SPFx v1.3.x+, such as application customizers & field customizers as they can also use React & Fluent UI React so they'd also be included in this doc, correct?